FaqGen/docker_compose/amd/gpu/rocm/compose.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  faqgen-tgi-service:
+    image: ghcr.io/huggingface/text-generation-inference:2.3.1-rocm
+    container_name: faggen-tgi-service
+    ports:
+      - "${FAQGEN_TGI_SERVICE_PORT:-8008}:80"
+    environment:
+      no_proxy: ${no_proxy}
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      TGI_LLM_ENDPOINT: "http://${HOST_IP}:${FAQGEN_TGI_SERVICE_PORT}"
+      HUGGINGFACEHUB_API_TOKEN: ${FAQGEN_HUGGINGFACEHUB_API_TOKEN}
+      HUGGING_FACE_HUB_TOKEN: ${FAQGEN_HUGGINGFACEHUB_API_TOKEN}
+    volumes:
+      - "/var/opea/faqgen-service/data:/data"
+    shm_size: 1g
+    devices:
+      - /dev/kfd:/dev/kfd
+      - /dev/dri/${DOCSUM_CARD_ID}:/dev/dri/${DOCSUM_CARD_ID}
+      - /dev/dri/${DOCSUM_RENDER_ID}:/dev/dri/${DOCSUM_RENDER_ID}
+    cap_add:
+      - SYS_PTRACE
+    group_add:
+      - video
+    security_opt:
+      - seccomp:unconfined
+    ipc: host
+    command: --model-id ${FAQGEN_LLM_MODEL_ID}
+  faqgen-llm-server:
+    image: ${REGISTRY:-opea}/llm-faqgen-tgi:latest
```

Comment:
Please do not freeze the image tag, we use `ci` for CI test.

Reply:
refer to this, https://github.com/opea-project/GenAIExamples/blob/82f418bf3439672ce570d05cbe2a1ca257b3d6b8/DocSum/docker_compose/intel/hpu/gaudi/compose.yaml#L28

Reply:
@artem-astafev would you please make the change @chensuyue suggested as soon as possible ... we are fast closing in on the code freeze deadline.

Reply:
Sure, thank you for the information, 

I will rollback changes for tags in compose, and will continue work on functional tests, 

there are still some issues with docker build part.

Reply:
I've reverted Tags for compose, this id done